### PR TITLE
Modified footer contact form

### PR DESCRIPTION
**Pull request by Chaitalishetty**

### CODE REVIEW

**Comment by smaranjitghose** [comment]
The vertical space between each form element should be the same as it is present in the hosted version of the website. Please rectify that

**Comment by Chaitalishetty** [comment]
I have changed the animation that is responsive for all the screens. Is it fine or should i make any changes?

File: assets/css/style.css [hunk 1]
```diff
@@ -807,6 +807,26 @@ form{
...
+
+.footer hr
+{
+    background-color: rgb(37, 51, 46);
+    height: 1.5px;
+}
+
+
+
+>>>>>>> f16ce2f2466ece18f2c52bed883fb862cd57603b
```
Comment (smaranjitghose) [inline assets/css/style.css:829]: Remove these. Please be mindful while merging

File: assets/css/style.css [hunk 2]
```diff
@@ -807,6 +807,26 @@ form{
 .form-style-6 input[type="button"]:hover{
 	background: rgb(74,71,163,0.8);
 }
+<<<<<<< HEAD
+=======
```
Comment (smaranjitghose) [inline assets/css/style.css:811]: Remove these. Please be mindful while merging

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Responsiveness Issue still persists on multiple screen sizes:

|          |         |  
| ------|------|
|     Respons_7     |  Respons_6     |          
|    Respons_3      |     Respons_2   |              
|Respons_4 | Respons_5   | 
| Respons_1    | |

**Comment by smaranjitghose** [comment]
Raise a new PR for this

**Pull request closed without merging**